backend/backend.go

```diff
@@ -555,6 +557,38 @@ func (backend *Backend) Coin(code coinpkg.Code) (coinpkg.Coin, error) {
 	return coin, nil
 }
 
+func (backend *Backend) pollETHBalances() {
+	timer := time.After(0)
+
+	for {
+		<-timer
+		if err := backend.updateETHAccounts(); err != nil {
+			backend.log.WithError(err).Error("could not update ETH accounts")
+		}
+		timer = time.After(eth.PollInterval)
+	}
+}
+
+func (backend *Backend) updateETHAccounts() error {
+	defer backend.accountsAndKeystoreLock.RLock()()
+	backend.log.Debug("Updating ETH accounts balances")
+
+	ethAccounts := make([]*eth.Account, 0, len(backend.accounts))
+	for _, account := range backend.accounts {
+		ethAccount, ok := account.(*eth.Account)
+		if ok {
+			ethAccounts = append(ethAccounts, ethAccount)
+		}
+
+	}
+
+	etherScanClient := etherscan.NewEtherScan("1", backend.etherScanHTTPClient)
+	if backend.Testing() {
```

Comment:
As mentioned before, this is not a robust check - we used to have multiple testnets with different chainIDs in the past (and might again in the future). 

eth.coin.ChainID() contains the chainID, could just use that?